packages/components/webpack.config.js

```diff
@@ -34,6 +34,7 @@ module.exports = {
         'mobile-wrapper'                : 'Components/mobile-wrapper',
         modal                           : 'Components/modal',
         money                           : 'Components/money',
+        'relative-datepicker'           : 'Components/relative-datepicker',
```

Comment:
Please use the alphabetic order when inserting items in this list.

Reply:
Done